tests/playback.js

```diff
@@ -117,6 +129,16 @@ queue_next_packet = function () {
 var bytes_processed = 0;
 
 do_packet = function () {
+    // Avoid having an excessive queue buildup
+    if (rfb._flushing) {
```

Comment:
> Yep, woke up in the middle of the night last night and realized what you were saying :-P. I'd still prefer if we could only turn it on "performance" mode though, since I use recordings to debug issues in the realitime mode, and sometimes timing matters there.
> 

I suppose it's possible. But I wouldn't have too much faith in the timing system. The browsers do not respect the timeout value given.

> Honestly, though, sometimes checking buffer management is what I want to do with the performance test. Could we maybe just have it behind an option (I've used this code to fix GC-related performance issues in our buffer management, for instance)?

That sounds like something that a more specific unit test is better suited for. This is a "whole system" after all. You never quite know what parts might be included.